engine/api/ui.go

```diff
@@ -0,0 +1,105 @@
+package api
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/ovh/cds/engine/api/application"
+	"github.com/ovh/cds/engine/api/metrics"
+	"github.com/ovh/cds/engine/api/navbar"
+	"github.com/ovh/cds/engine/api/project"
+	"github.com/ovh/cds/engine/api/repositoriesmanager"
+	"github.com/ovh/cds/engine/api/workflow"
+	"github.com/ovh/cds/engine/service"
+	"github.com/ovh/cds/sdk"
+)
+
+func (api *API) getNavbarHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		data, err := navbar.LoadNavbarData(api.mustDB(), api.Cache, getUser(ctx))
+		if err != nil {
+			return sdk.WrapError(err, "getNavbarHandler")
+		}
+		return service.WriteJSON(w, data, http.StatusOK)
+	}
+}
+
+func (api *API) getApplicationOverviewHandler() service.Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		vars := mux.Vars(r)
+		key := vars["key"]
+		appName := vars["permApplicationName"]
+
+		p, errP := project.Load(api.mustDB(), api.Cache, key, getUser(ctx))
+		if errP != nil {
+			return sdk.WrapError(errP, "getApplicationOverviewHandler> unable to load project")
```

Comment:
Why not simply "err" in all this func ?

Reply:
to avoid shadowing err, we recommend to use err only on online check